example/app/App.tsx

```diff
@@ -19,11 +19,16 @@ const requestPermissions = async () => PermissionsAndroid.requestMultiple(PERMIS
 
 export const App = () => {
   const [qrCode, setQrCode] = useState<string>()
-  const [mdt, setMdt] = useState<ReturnType<typeof mdocDataTransfer.instance>>()
+  const [mdt, setMdt] = useState<Awaited<ReturnType<typeof mdocDataTransfer.instance>>>()
 
   useMdocDataTransferShutdownOnUnmount()
 
-  const start = () => setMdt(mdocDataTransfer.instance('ma_service'))
+  const start = async () => {
+    const m = await mdocDataTransfer.instance('ma_service', [
+      'MIIBxDCCAWugAwIBAgIQTn6+uOu5roCD/2HCc8v93TAKBggqhkjOPQQDAjAdMQ4wDAYDVQQDEwVBbmltbzELMAkGA1UEBhMCTkwwHhcNMjQwMzI0MTEzODM2WhcNMjgwMzI0MTEzODM2WjAdMQ4wDAYDVQQDEwVBbmltbzELMAkGA1UEBhMCTkwwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQC/YyBpcRQX8ZXpHfra1TNdSbS7qzgHYHJ3msbIr8TJLPNZI8Ul8zJlFdQVIVls5+5ClCbN+J9FUvhPGs4AzA+o4GMMIGJMB0GA1UdDgQWBBQv3zBo1i/1CfEgdvkIWDGO9lS1SzAOBgNVHQ8BAf8EBAMCAQYwIQYDVR0SBBowGIYWaHR0cHM6Ly9mdW5rZS5hbmltby5pZDASBgNVHRMBAf8ECDAGAQH/AgEAMCEGA1UdHwQaMBgwFqAUoBKGEGh0dHBzOi8vYW5pbW8uaWQwCgYIKoZIzj0EAwIDRwAwRAIgWPX8NGiW1G7PwK+K7NVxd3h8DdZpZbB/u8GsXhvr4mACIF+DwCalXfWdu5tFM3iOJytQDPNOkFSnvE6eQK3x7BLG',
+    ])
+    setMdt(m)
```

Comment:
Let's use options object for clarity?

```suggestion
    const m = await mdocDataTransfer.instance('ma_service', {
     trustedCertificates: [
      'MIIBxDCCAWugAwIBAgIQTn6+uOu5roCD/2HCc8v93TAKBggqhkjOPQQDAjAdMQ4wDAYDVQQDEwVBbmltbzELMAkGA1UEBhMCTkwwHhcNMjQwMzI0MTEzODM2WhcNMjgwMzI0MTEzODM2WjAdMQ4wDAYDVQQDEwVBbmltbzELMAkGA1UEBhMCTkwwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQC/YyBpcRQX8ZXpHfra1TNdSbS7qzgHYHJ3msbIr8TJLPNZI8Ul8zJlFdQVIVls5+5ClCbN+J9FUvhPGs4AzA+o4GMMIGJMB0GA1UdDgQWBBQv3zBo1i/1CfEgdvkIWDGO9lS1SzAOBgNVHQ8BAf8EBAMCAQYwIQYDVR0SBBowGIYWaHR0cHM6Ly9mdW5rZS5hbmltby5pZDASBgNVHRMBAf8ECDAGAQH/AgEAMCEGA1UdHwQaMBgwFqAUoBKGEGh0dHBzOi8vYW5pbW8uaWQwCgYIKoZIzj0EAwIDRwAwRAIgWPX8NGiW1G7PwK+K7NVxd3h8DdZpZbB/u8GsXhvr4mACIF+DwCalXfWdu5tFM3iOJytQDPNOkFSnvE6eQK3x7BLG',
    ]})
    setMdt(m)
```